style.css

```diff
@@ -1,5 +1,6 @@
 body {
-    background-color: aquamarine;
+    background-color: rgb(203, 24, 209);
+    border: solid 3px rgb(95, 24, 209);
```

Comment:
man, is not fucsia, should be "pink" color

Reply:
Ok, chenge yet